dags/process_elife_zip_dag.py

```diff
@@ -66,6 +68,32 @@ def get_article_from_previous_task(context: dict) -> ElementTree:
     return etree.parse(BytesIO(article_bytes))
 
 
+def convert_image_in_s3_to_jpeg(key) -> str:
+    s3 = get_aws_connection('s3')
+    with TemporaryFile(dir=TEMP_DIRECTORY) as temp_tiff_file:
+        s3.download_fileobj(
+            Bucket=DESTINATION_BUCKET,
+            Key=key,
+            Fileobj=temp_tiff_file
+        )
+        temp_tiff_file.seek(0)
+
+        logger.info('converting %s to jpeg', key)
+
+        temp_jpeg = BytesIO()
+        with Image(file=temp_tiff_file) as img:
+            img.format = 'jpeg'
+            img.save(file=temp_jpeg)
+
+        key = re.sub(r'\.\w+$', '.jpg', key)
+        s3.put_object(
+            Bucket=DESTINATION_BUCKET,
+            Key=key,
+            Body=temp_jpeg.getvalue()
+        )
```

Comment:
would introduce a log at the start and at the end of the method, that can help have a rough understanding of start and stop time for each section; when they execute in parallel it can be a bit of a mess to identify which process is doing what.